go.mod

```diff
@@ -15,5 +15,6 @@ require (
 	go.uber.org/automaxprocs v1.4.0
 	google.golang.org/grpc v1.38.0
 	google.golang.org/protobuf v1.26.0
+	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
```

Comment:
Usually , a new item in `go.mod` requires a new item in `LICENSE` file

Reply:
Thx, have added.

Reply:
Is this for satellite runtime? If not, we don't need the license.

Reply:
> Is this for satellite runtime? If not, we don't need the license.

It's built into the binary, although we only use it to generate docs

Reply:
> > Is this for satellite runtime? If not, we don't need the license.
> 
> It's built into the binary, although we only use it to generate docs

Also, the other PR #42  will use this at runtime, I think it's ok to have this license for now, no matter which PR gets merged first.

Reply:
Should I need to use the YAML framework v2 version? I have checked, the result basically the same.

Reply:
> Should I need to use the YAML framework v2 version? I have checked, the result basically the same.

Let's use v3, I think newer version is preferable